appstudio-utils/util-scripts/cosign-verify-attestation.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I always try to use the unofficial [Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) in my bash scripts. Maybe it could be beneficial for us to start using it here?

Reply:
Yeah, I don't see any major downsides.